src/uu/tr/src/operation.rs

```diff
@@ -319,13 +329,31 @@ impl Sequence {
             recognize(many_m_n(1, 3, one_of("01234567"))),
             |out: &[u8]| {
                 let str_to_parse = std::str::from_utf8(out).unwrap();
+                match u8::from_str_radix(str_to_parse, 8) {
+                    Ok(ue) => Some(ue),
+                    Err(_pa) => None,
+                }
```

Comment:
You can simplify this and use `ok()` to turn the `Result` into an `Option`:
```suggestion
                u8::from_str_radix(str_to_parse, 8).ok()
```

Reply:
i wonder if there is a clippy warning for this?!

Reply:
@sylvestre I'm not aware of such a lint.

Reply:
Yes, you are right. I just copied it and removed the previous TODO comment, so I didn't notice that this match is redundant.

Reply:
@samueltardieu you might be interested by this :)

Reply:
See https://github.com/rust-lang/rust-clippy/pull/13740